library/std/src/path.rs

```diff
@@ -2405,6 +2418,18 @@ impl Path {
     ///
     /// This is a convenience function that coerces errors to false. If you want to
     /// check errors, call [`fs::metadata`].
+    ///
+    /// An example of proper error handling.
+    /// ```no_run
+    /// use std::path::Path;
+    /// use std::io::ErrorKind;
+    /// use std::fs::metadata;
+    /// let exists = metadata(Path::new("does_not_exist.txt"))
```

Comment:
The function is canonically invoked as `fs::metadata(...)`, like seen in https://doc.rust-lang.org/1.51.0/std/fs/fn.metadata.html#examples.

```suggestion
    /// let exists = fs::metadata(Path::new("does_not_exist.txt"))
```